exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -18,24 +18,37 @@ OvenTime
 Define the `RemainingOvenTime()` function that takes the actual minutes the lasagna has been in the oven as a parameter and returns how many minutes the lasagna still has to remain in the oven, based on the expected oven time in minutes from the previous task.
 
 ```go
+func RemainingOvenTime(actual int) int {
+    // TODO
+}
+
 RemainingOvenTime(30)
 // Output: 10
 ```
 
 ## 3. Calculate the preparation time in minutes
 
-Define the `PreparationTime()` function that takes the number of layers you added to the lasagna as a parameter and returns how many minutes you spent preparing the lasagna, assuming each layer takes you 2 minutes to prepare.
+Define the `PreparationTime(layers int) int` function that takes the number of layers you added to the lasagna as a parameter and returns how many minutes you spent preparing the lasagna, assuming each layer takes you 2 minutes to prepare.
```

Comment:
This seems likely to get out of sync if we ever update the method signature for any reason.

Just remove any type information and leave the function name only.

```suggestion
Define the `PreparationTime` function that takes the number of layers you added to the lasagna as a parameter and returns how many minutes you spent preparing the lasagna, assuming each layer takes you 2 minutes to prepare.
```